### ISSUE TITLE
Update all the docs once CoreFX and CoreCLR are merged into dotnet/runtime

### ISSUE THREAD

**Issue by adamsitnik** [dotnet/performance#990]

**Comment by ts2do** [comment]
Is there any estimate on when this will be ready? Thanks.

**Comment by adamsitnik** [comment]
I am working on this, currently we are blocked by https://github.com/dotnet/runtime/issues/1523
